### ISSUE TITLE
I can't start the GPU 3060 

### ISSUE THREAD

**Issue by nofacedeepfake** [abdallahdib/NextFace#15]
I can't start the GPU 3060

**Comment by abdallahdib** [comment]
can u plz elaborate more on ur problem so that i can try to help

**Comment by 51gohome** [comment]
It may be caused by redner: https://github.com/BachiLi/redner/issues/169. Unfortunately, the issue has not been fixed.

**Comment by JamesBDH** [comment]
I've hit this same issue using a 3080.

`RuntimeError: Function "_rtpModelUpdate" caught exception: Encountered a CUDA error: radix_sort_temp_size -> cub::DeviceRadixSort::SortPairs returned (8): invalid device function`

**Comment by abdallahdib** [comment]
I think redner needs to be compiled for cuda 11. meantime try to get pytorch with cuda 10 support. plz execute the commands in **INSTALL** file which will create a compatible environment for u. Hope this help.

**Comment by JamesBDH** [comment]
Yeah, looks like 3000-series GPUs are incompatible with Cuda 10, so unless Redner gets Cuda 11 support, it's not going to be possible to run NextFace on 3000-series GPUs.

**Comment by abdallahdib** [comment]
there is no reason for not being able to install pytorch 1.3 with cudatoolkit 10.1 on ur gpu 3000. 
running this command should work on ur machine : 

**conda install pytorch==1.3.1 torchvision==0.4.2 cudatoolkit=10.1 -c pytorch**

u can always run NextFace on Cpu (change device to cpu in **optimConfig.ini**)

**Comment by JamesBDH** [comment]
Nope, that doesn't work, I get the error I mentioned before when running the reconstruction. Same issue brought up here: https://github.com/BachiLi/redner/issues/169. Will try with the CPU though.

**Comment by abdallahdib** [comment]
ok i see 
does **torch.version.cuda** returns 10.1 ?

**Comment by arstropica** [comment]
The first Cuda version to recognize GTX 30xx cards was 11.1
https://docs.nvidia.com/cuda/archive/11.1.0/cuda-toolkit-release-notes/index.html#cuda-general-new-features

**Comment by huhai463127310** [comment]
> I think redner needs to be compiled for cuda 11. meantime try to get pytorch with cuda 10 support. plz execute the commands in **INSTALL** file which will create a compatible environment for u. Hope this help.

I have compile redner from source for cuda 11. But the issue didn't resolve. My GPU is 3080 Laptop.

**Comment by huhai463127310** [comment]
> The first Cuda version to recognize GTX 30xx cards was 11.1 https://docs.nvidia.com/cuda/archive/11.1.0/cuda-toolkit-release-notes/index.html#cuda-general-new-features

Colud you run the code under cuda 11.1 for RTX 30 series ?

**Comment by thekesslereffect** [comment]
also having this same issue with an RTX 3080Ti :( cpu looks like it's running fine.

**Comment by BbChip0103** [comment]
save issue here for A100

**Comment by Sarayel** [comment]
not only redner but also pytorch from version 1.7 is compatibile with CUDA 11.x so as long as repo won't be updated to those, it won't run on ampere arch.

**Comment by arstropica** [comment]
> @arstropica
> 
> > The first Cuda version to recognize GTX 30xx cards was 11.1 https://docs.nvidia.com/cuda/archive/11.1.0/cuda-toolkit-release-notes/index.html#cuda-general-new-features
> 
> Could you run the code under cuda 11.1 for RTX 30 series ?

No.  Redner uses Optix 6.5 which will never be compatible with RTX30xx.
https://forums.developer.nvidia.com/t/optix-6-5-prime-samples-fail-with-rtx-3080/177078

**Comment by yosun** [comment]
hm hardware requirements ought to be listed if RTX30X0 is not supported

what is the hardware you are running on?!

**Comment by arstropica** [comment]
> hm hardware requirements ought to be listed if RTX30X0 is not supported
> 
> what is the hardware you are running on?!

I ended up changing my Optix config to run on cpu and also updated my torch version to 1.12.1.  Shame because I have an RTX3090, but no other way to go.

**Comment by younge529** [comment]
> > hm hardware requirements ought to be listed if RTX30X0 is not supported
> > what is the hardware you are running on?!
> 
> I ended up changing my Optix config to run on cpu and also updated my torch version to 1.12.1. Shame because I have an RTX3090, but no other way to go.

Hello. can you tell me how i can run optix on cpu?

**Comment by arstropica** [comment]
> Hello. can you tell me how i can run optix on cpu?

Try my forked branch @: https://github.com/arstropica/NextFace/tree/RTX30XX

Note the [changes](https://github.com/arstropica/NextFace/blob/RTX30XX/INSTALL) to the INSTALL doc.
